lib/Backends/CPU/CPUFunction.h

```diff
@@ -18,6 +18,7 @@
 
 #include "GlowJIT.h"
 
+#include "glow/Backends/BackendUtils.h"
```

Comment:
Why is it necessary to include here? Can you just include in `CPUFunction.cpp` instead?

Reply:
Ah whoops I totally missed what was going on here, ignore me 😀